core/frontend/src/IModelApp.ts

```diff
@@ -271,6 +276,9 @@ export class IModelApp {
    */
   public static readonly telemetry: TelemetryManager = new TelemetryManager();
 
+  /** @alpha */
```

Comment:
Why is `IModelApp.extensionAdmin` `@alpha` when all the other APIs are `@beta`?

Reply:
updated to @beta